<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGuiAppClass</class>
 <widget class="QMainWindow" name="QtGuiAppClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtGuiApp</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>600</width>
      <height>60</height>
     </rect>
    </property>
    <property name="title">
     <string>Altitude</string>
    </property>
    <widget class="QLCDNumber" name="LcdAltitude">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>320</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="CheckAltitude">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Altitude</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="TextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>600</width>
      <height>160</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>600</width>
      <height>90</height>
     </rect>
    </property>
    <property name="title">
     <string>Throttle</string>
    </property>
    <widget class="QLCDNumber" name="LcdThrottle">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>320</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="CheckThrottle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Throttle</string>
     </property>
    </widget>
    <widget class="QSlider" name="ControlThrottle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>580</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>620</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtGuiApp.qrc"/>
 </resources>
 <connections/>
</ui>
